pretixprint/app/src/main/java/eu/pretix/pretixprint/print/ESCPOSRenderer.kt

```diff
@@ -634,4 +668,30 @@ class ESCPOSRenderer(private val receipt: JSONObject, private val charsPerLine:
         out.add(durationOn.toByte())
         out.add(durationOff.toByte())
     }
+
+    fun renderTestPage(): ByteArray {
+        out.clear()
+        init()
+        if (dialect == Dialect.EpsonDefault) {
+            characterCodeTable(CharacterCodeTable.WPC1252.codeTable)
+            internationalCharacterSet(InternationalCharacterSet.Germany.country)
+        } else if (dialect == Dialect.Sunmi) {
+            selectKanjiCharacterMode()
+            selectKanjiCharacterCodeSystem(-1)
+        }
+        qr("TEST COMPLETED", 6)
+        newline()
+        text("German: äöüÄÖÜß Euro sign: €", align = LEFT)
```

Comment:
On my Epson TM-m30 both character code tables `WPC1252` / `UserDefined2` produce not quite successful results - this may confuse customers too:

`WPC1252`: German: äöüÄÖÜß Euro sign: ¬  
`UserDefined2`: German:         Euro sign:

Reply:
Yeah, we want WPC1252 on Epson. I'll remove the euro sign again, pretixPOS does not use it right now.